### PR TITLE
mgr/volumes: add arg to fs volume create for mds daemons placement

### PR DESCRIPTION
**Pull request by Daniel-Pivonka**
add placement arg for mds daemons created by fs volume create
add ability to parse placementspec from strings

Fixes: https://tracker.ceph.com/issues/43835

Signed-off-by: Daniel-Pivonka dpivonka@redhat.com

### CODE REVIEW

File: src/pybind/mgr/volumes/module.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         },
         {
             'cmd': 'fs volume create '
-                   'name=name,type=CephString ',
+                   'name=name,type=CephString '
+                   'name=mds_placement,type=CephString,req=false ',
```
Comment (liewegas) [inline src/pybind/mgr/volumes/module.py]: nit: let's call this parameter ``placement`` everywhere for consistency?

File: src/pybind/mgr/volumes/module.py [hunk 2]
```diff
@@ -242,7 +243,8 @@ def handle_command(self, inbuf, cmd):
 
     def _cmd_fs_volume_create(self, inbuf, cmd):
         vol_id = cmd['name']
-        return self.vc.create_fs_volume(vol_id)
+        placement = cmd.get('mds_placement', None)
```
Comment (liewegas) [inline src/pybind/mgr/volumes/module.py]: ``placement``?

**Comment by batrick** [comment]
This PR broke vstart and probably teuthology QA: https://tracker.ceph.com/issues/44244

**Comment by batrick** [comment]
cc @ajarr @vshankar

**Pull request merged**